lib/rules/id-blacklist.js

```diff
@@ -101,21 +104,21 @@ module.exports = {
                         // Report AssignmentExpressions only if they are the left side of the assignment
                     } else if (effectiveParent.type === "AssignmentExpression" &&
                         (effectiveParent.right.type !== "MemberExpression" ||
-                        effectiveParent.left.type === "MemberExpression" &&
-                        effectiveParent.left.property.name === node.name)) {
+                            effectiveParent.left.type === "MemberExpression" &&
+                            effectiveParent.left.property.name === node.name)) {
                         if (isInvalid(name)) {
                             report(node);
                         }
                     }
 
-                // Properties have their own rules
+                    // Properties have their own rules
```

Comment:
Mind undoing this? This is referring to the follow block and I think it's clearer to have it indented as it was.